go/fn/testhelpers/golden.go

```diff
@@ -15,18 +29,18 @@ import (
 )
 
 // RunGoldenTests provides the test infra to run golden test.
-// - "basedir" should be the parent directory, under where the sub-directories contains test data.
-//   For example, the "testdata" is the basedir. It contains two cases "test1" and "test2"
-//   └── testdata
-//       └── test1
-//           ├── _expected.yaml
-//           ├── _fnconfig.yaml
-//           └── resources.yaml
-//       └── test2
-//           ├── _expected.yaml
-//           ├── _fnconfig.yaml
-//           └── resources.yaml
-// - "krmFunction" should be your ResourceListProcessor implementation.
+//   - "basedir" should be the parent directory, under where the sub-directories contains test data.
+//     For example, the "testdata" is the basedir. It contains two cases "test1" and "test2"
+//     └── testdata
+//     └── test1
+//     ├── _expected.yaml
+//     ├── _fnconfig.yaml
+//     └── resources.yaml
+//     └── test2
+//     ├── _expected.yaml
+//     ├── _fnconfig.yaml
+//     └── resources.yaml
```

Comment:
The updated one messed up the indentation making the example useless :)

Reply:
Ah, this is a very interesting behavior of gofmt comment formatting. It looks like the leading dash in the first comment line (`- "basedir" should be ... `) is treated as invalid indentation, so gofmt reformatted the comments.

Fixed and thank you!

Reply:
Interesting. TIL :)